### PR TITLE
added first draft of contributor guidelines

### PR DESCRIPTION
**Pull request by JessicaS11**
I added a new file with contributor guidelines. Feedback solicited on the document. Thanks in advance for your constructive suggestions and edits. : )

### CODE REVIEW

File: contribution_guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Questions and Help
+------------------
+
+Please do not create issues to ask for help. A faster way to reach the community is through our Science/ICESat-2 subcategory on the `Pangeo discourse <https://discourse.pangeo.io/c/science/icesat-2/16>`_ page. We are excited to have you join an existing conversation or start a new post! Please note that a github login is required to post on the discourse page.
+
+Other Resources
+^^^^^^^^^^^^^^^
+- We are currently working on compiling a list of resources for working with ICESat-2 data (in the meantime, check out the `2019 ICESat-2 Hackweek repo<https://github.com/ICESAT-2HackWeek>`_ and `consider applying for this year's Hackweek<https://icesat-2hackweek.github.io/>`) 
+- A great set of `interactive tutorials<https://learngitbranching.js.org>`_ for learning and practicing using git
+- Let us know about the helpful tools you've found!
```
Comment (fperez) [inline contribution_guidelines.rst]: "... by posting on our discourse forum as indicated above" ?

File: contribution_guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+Adding Examples
+---------------
+We are delighted you'd like to contribute your icepyx/ICESat-2 example! Examples may be in the form of executable scripts or interactive Jupyter Notebooks. Please make sure that each example has a descriptive name so someone not familiar with your project understands its general behavior. Fully working examples should be submitted using a pull request to the "examples" branch, following the steps outlined below for **Contributing Code**.
+
+Question - Do we want to strongly suggest all examples submitted be in python? Are they required to use icepyx, or just ICESat-2 data?
```
Comment (fperez) [inline contribution_guidelines.rst]: I think in this repo we should limit it to icepyx. Having a matlab example using third party tools in this repo would be weird...

We could, if there's enough interest, start a companion repo for illustrative examples that use any other toolchain, in the top-level icesat2py org (though the 'py' is a bit of a hint that we might want to shed then :)

File: contribution_guidelines.rst (hunk 1 above)
Comment (JessicaS11) [reply contribution_guidelines.rst]: Added this in with a link to the section.

File: contribution_guidelines.rst (hunk 2 above)
Comment (JessicaS11) [reply contribution_guidelines.rst]: I agree - I think a companion repo, should the need arise, is a great way to make sure anyone working with ICESat-2 data can contribute/participate. I've modified the text to just say icepyx examples.

**Pull request merged**